src/popup.js

```diff
@@ -10,7 +10,7 @@ import events from './helpers/events';
   const addHorizontalRule = document.querySelector('#horizontal-rule');
   const codeFormatLanguage = document.querySelector('#code-lang');
   const links = document.querySelectorAll('a');
-  const UDEMY_HOST = 'www.udemy.com';
+  const UDEMY_HOST_REGEXP = /(?:[a-zA-Z0-9\-]{0,63}\.)?udemy\.com/;
```

Comment:
i think more generalized regex would be better `/(?:.*?)\.udemy\.com/i`. similar to #7